index.html

```diff
@@ -9,398 +9,165 @@
+    <section class="contact-form" id="contact-form">
+      <div class="contact-form__wrapper">
+        <h2 class="contant-form__header">Contact me</h2>
+        <p class="contact_form__paragraph">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I'd love
+          to help with it
+        </p>
+
+        <form class="contact_me__form" action="https://formspree.io/f/meqdygwl" method="POST" id="form">
+          <label for="name" class="form-label">Enter your name</label>
+          <input type="text" name="name" maxlength="50" placeholder="Enter your name" required class="field-data" /><br><br>
+
+          <label for="email" class="form-label"></label>
+          <input type="email" name="email" required placeholder="Enter your email" class="field-data" /><br><br>
+
+          <label for="message" class="form-label"></label>
+          <textarea name="message" maxlength="500" required placeholder="Write your message here" class="field-data2"></textarea>
+          <div class="error-message">
+            <span id="error-message" class="hidden">Testing error message</span>
           </div>
-          <ul>
-            <li>Canopy</li>
-            <li><span class="dot"></span> Back End Dev</li>
-            <li><span class="dot"></span>2015</li>
-          </ul>
-        </div>
-        <div class="modal--body">
-          <img src="" alt="" class="modal--image">
-        </div>
-        <div class="modal--footer">
-          <p class="modal--description"></p>
-          <div class="modal--buttons-techs-wrapper">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-            <div>
-              <button class="btn btn-outline"><a href="#" class="modal--see-live">See Live <img src="./assets/images/see-live.svg" alt="button"></a></button>
-              <button class="btn btn-outline"><a href="#" class="modal--see-source">See Source <img src="./assets/images/github.svg" alt="button"></a></button>
-            </div>
+          
+          <button>Get in Touch</button>
+          <div class="contact-form__black-bar">
+            <div></div>
           </div>
+        </form>
+      </div>
+    </section>
+
+    <section class="mobile-menu hidden" id="mobile-menu">
+      <div class="mobile-menu__close">
+        <img src="img/close-icon.png" id="close-img" alt="close-icon">
+      </div>
+      <ul>
+        <li><a href="#works-wrapper">Portfolio</a></li>
+        <li><a href="#about-myself">About</a></li>
+        <li><a href="#contact-form">Contact</a></li>
+      </ul>
+    </section>
+
+    <section class="modal hidden" id="modal">
+      <div class="modal__wrapper">
+        <div class="modal__close" id="close-modal"><img src="img/close-icon.png" alt="close-icon"></div>
+        <div class="modal__content" id="modal-content">
         </div>
       </div>
-     </div>
```

Comment:
#### Good job on displaying the modal @fauziyahzeeyah. I tried to open your project both locally and live I have seen some minor details needed to be fixed. 
- [ ] Your images are not displayed, which makes your modal a bit off compared to the Figma design.
- [ ] I believe your design could be more responsive, on a screen size of `800px` I see your footer text is overflowing. Please address this for a better-looking modal. 

### Your popup on a screen size of `800px`
![Screenshot 2022-12-15 084416](https://user-images.githubusercontent.com/69077061/209213768-8222d7ae-8439-4511-be82-db165e3ef8c4.png)